pico/tools/src/main/java/io/helidon/pico/tools/ModuleInfoDescriptorSupport.java

```diff
@@ -157,6 +157,8 @@ static ModuleInfoDescriptor create(String moduleInfo,
                         }
                     }
                     descriptor.addItem(provides.build());
+                } else if (line.startsWith("opens ")) {
+                    // follow up issue
```

Comment:
Can we leave the issue number in comment for later clarity?

Reply:
Added to PR description